src/script/src/manager.rs

```diff
@@ -149,9 +158,11 @@ mod tests {
             .unwrap();
         catalog_manager.start().await.unwrap();
 
+        let schema = "test";
```

Comment:
Consider using different names for the script and schema. It should help discover some potential problems. Now they are both `test`.

Reply:
That's right